.github/workflows/trivy_scan.yml

```diff
@@ -0,0 +1,77 @@
+name: Scan pmm server and client docker images
+
+on:
+  workflow_dispatch:
+    inputs:
+      server_image:
+        description: "pmm-server docker image"
+        required: true
+        type: string
+      client_image:
+        description: "pmm-client docker image"
+        required: true
+        type: string
+      sha:
+        description: "SHA (leave empty if running manually, default - 'null')"
```

Comment:
```suggestion
        description: "SHA (leave empty if running manually)"
```
I'd better not use 'null' as a default string value, as it may confuse people. Isn't the default an empty string if you don't pass the value? Moreover, I don't see the use of the `default` keywork here.

Reply:
I copied this section from the workflow located at:
https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/.github/workflows/pmm2-testsuite.yml

After you brought it up, I too believe that the default value is an empty string rather than 'null'.

What do you think, @yurkovychv ?

Reply:
I believe they are unrelated, i.e. we could get this one right without touching the other, no?